### PR TITLE
ape: init at 6.7-131003

### PR DESCRIPTION
**Pull request by yrashk**
###### Motivation for this change

Attempto APE is not in Nixpkgs

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by yrashk** [comment]
Any concerns about these derivations?

**Comment by yrashk** [comment]
Whom should I ping to review this PR?

File: pkgs/applications/misc/ape/apeclex.nix [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{ stdenv, attemptoClex, callPackage }:
+
+callPackage ./. {
+  pname = "ape-clex";
+  lexicon = "${attemptoClex}/clex_lexicon.pl";
+  description = "Parser for Attempto Controlled English (ACE) with a large lexicon (~100,000 entries)";
```
Comment (cleverca22) [inline pkgs/applications/misc/ape/apeclex.nix]: why are these values not in default.nix?

Comment (yrashk) [reply pkgs/applications/misc/ape/apeclex.nix]: They are -- there are just two variations of the package, with the small lexicon and a larger one. Smaller lexicon is covered by LGPL, however, the larger one is GPL.

**Comment by yrashk** [comment]
Closed for inactivity.

**Pull request closed without merging**